src/storage/txn/scheduler.rs

```diff
@@ -820,12 +832,14 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             + statistics.cf_statistics(CF_WRITE).flow_stats.read_bytes;
         sample.add_read_bytes(read_bytes);
         let quota_delay = quota_limiter.consume_sample(sample, true).await;
+        let mut total_throttle_duration =
+            TrackGuard::new(tracker, |m| &mut m.scheduler_throttle_nanos);
         if !quota_delay.is_zero() {
+            total_throttle_duration.value += quota_delay.as_nanos() as u64;
```

Comment:
I'm curious about where we really execute this delay? Can we confirm that the number here is the actual delay time? Is it possible that Runtime will call for a longer period of time? 

For example, in the flow_controller department below, the specific scheduling time can be calculated explicitly.

Reply:
Good idea. I believe they are approximately the same. But it's absolutely more accurate to record it ourselves.